direct/io-hadoop/src/test/resources/log4j.properties

```diff
@@ -1,6 +1,6 @@
 # Root logger option
-log4j.rootLogger=INFO, stdout
-#log4j.rootLogger=DEBUG, stdout
+#log4j.rootLogger=INFO, stdout
```

Comment:
minor: just clear this file (remove commented lines)

Reply:
These are there for debug. This PR should not have touched that. Fixed.